src/org/zaproxy/zap/extension/pscanrulesBeta/ZapAddOn.xml

```diff
@@ -1,17 +1,13 @@
 <zapaddon>
 	<name>Passive scanner rules (beta)</name>
-	<version>14</version>
+	<version>15</version>
 	<status>beta</status>
 	<description>The beta quality Passive Scanner rules</description>
 	<author>ZAP Dev Team</author>
 	<url/>
 	<changes>
 	<![CDATA[
-	Issue 2576: CSRFCountermeasures, remove unneeded Attack and Evidence messages.<br>
-	Issue 2574: CookieLooselyScopedScanner, remove attack field and update description.<br>
-	Support ignoring specified forms when checking for CSRF vulnerabilities.<br>
-	Correct the plugin ID of Absence of Anti-CSRF Tokens from 40014 to 10202.<br>
-	Issue 2860: Add further reference links.
+	Support security annotations for forms that dont need anti CSRF tokens.<br>
```

Comment:
I believe `Anti-CSRF` should be hyphenated, and `don't` should have an apostrophe.